client/src/components/Booking/ManageBookings.tsx

```diff
@@ -0,0 +1,180 @@
+import { useState } from 'react';
+import Booking from './Booking';
+import { makeStyles } from '@material-ui/styles';
+import { Grid, Paper, Typography } from '@material-ui/core';
+
+import { MuiPickersUtilsProvider, DatePicker } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { IRequest } from './IRequest';
+
+const useStyles = makeStyles({
+  root: {
+    justifyContent: 'space-between',
+  },
+  selectedBooking: {
+    padding: 10,
+  },
+  calendarGrid: {
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+  },
+  noBooking: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: '10px',
+    padding: '20px',
+  },
+});
+
+const ManageBookings = (): JSX.Element => {
+  const classes = useStyles();
+  const [date, changeDate] = useState<MaterialUiPickersDate>(new Date());
+  const [isSelectedDate, setIsSelectedDate] = useState<boolean>(false);
+
+  const [requests, setRequests] = useState<Array<IRequest>>([
+    {
+      status: 'pending',
+      sitterId: '1',
+      start: new Date(2021, 9, 8, 6),
+      end: new Date(2021, 9, 8, 6),
+    },
+    {
+      status: 'pending',
+      sitterId: '1',
+      start: new Date(2021, 9, 13, 6),
+      end: new Date(2021, 9, 13, 6),
+    },
+    {
+      status: 'accepted',
+      sitterId: '2',
+      start: new Date(2021, 10, 22, 14),
+      end: new Date(2021, 10, 22, 15),
+    },
+    {
+      status: 'declined',
+      sitterId: '1',
+      start: new Date(2021, 8, 1, 11),
+      end: new Date(2021, 8, 1, 15),
+    },
+    {
+      status: 'declined',
+      sitterId: '3',
+      start: new Date(2021, 10, 23, 14),
+      end: new Date(2021, 10, 23, 15),
+    },
+  ]);
```

Comment:
Good job using mocks, but let's move them to their own file (i.e. there is a `mock` folder for this as well).